sys-tune/source/impl/source.cpp

```diff
@@ -278,27 +278,24 @@ Source *OpenFile(const char *path) {
     if (R_FAILED(sdmc::OpenFile(&file, path)))
         return nullptr;
 
-    Source *source = nullptr;
-
     if (false) {}
 #ifdef WANT_MP3
     else if (strcasecmp(path + length - 4, ".mp3") == 0) {
-        source = new (std::nothrow) Mp3File(std::move(file));
+        return std::make_unique<Mp3File>(std::move(file));
```

Comment:
I don't believe we'll run out of memory but is the behavior of these identical?

Reply:
Basically yeah. Would you like me to revert it?

Reply:
no